yml/cdb_startrackerfast.yaml

```diff
@@ -0,0 +1,381 @@
+---
+name: cdb_startrackerfast
+"@id": "#cdb_startrackerfast"
+description: Consolidated Database for StarTrackerFast
+version:
+  current: "1.0.0"
+tables:
+- name: exposure
+  "@id": "#exposure"
+  primaryKey:
+  - "#exposure.exposure_id"
+  constraints:
+  - name: un_day_obs_seq_num
+    "@id": "#exposure.un_day_obs_seq_num"
+    "@type": Unique
+    description: Ensure day_obs plus seq_num is unique.
+    columns:
+    - "#exposure.day_obs"
+    - "#exposure.seq_num"
+  columns:
+  - name: exposure_id
+    "@id": "#exposure.exposure_id"
+    datatype: long
+    nullable: false
+    description: Unique identifier.
+  - name: day_obs
+    "@id": "#exposure.day_obs"
+    datatype: int
+    nullable: false
+    description: Day of observation.
+  - name: seq_num
+    "@id": "#exposure.seq_num"
+    datatype: int
+    nullable: false
+    description: Sequence number.
+  - name: s_ra
```

Comment:
The Header Service is generating these values for the LSST instruments. Is Rapid Analysis going to generate all of these for the Star Trackers?  If not, we should cut back to what is actually going to be written.

Reply:
I am pretty sure that that is the plan, but @mfisherlevine can confirm.

Reply:
Yes, that's right. And now looking at it, this one will be for the default constructed wcs, so we actually need a copy of these in the `quicklook` tables for the fitted wcs's as well. I assume `nullable` defaults to `true`, which perhaps doesn't matter much here, but definitely does need to be true for the quicklook tables, as the fits don't always succeed.

Reply:
Okay, I added ra and dec to the `quicklook` tables.

Reply:
It's not clear to me that we will need separate `exposure` and `exposure_quicklook` tables if they're both coming from the same place.  But it's also a good thing to have them similar to the other instruments.